src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisCacheWrapper.cs

```diff
@@ -11,21 +10,21 @@ namespace OrchardCore.Redis.Services;
 /// <remarks>
 /// This is done by not disposing the `RedisCache` instance which would otherwise close the redis connection.
 /// </remarks>
-public sealed class RedisCacheWrapper : IDistributedCache
+internal sealed class RedisCacheWrapper : IDistributedCache
```

Comment:
This is a breaking change, I might agree to set it `internal` if we have an extension method to add redis service

Reply:
It is, but this type should have never been public, it should be resolved through DI only.

Reply:
Forget my comment, while the class resides in the module